domain-analysis/info.json

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
can we remove the logo.xcf and medium.png?

Reply:
done